docs/apis/swagger-docs.yml

```diff
@@ -1029,4 +1033,235 @@ paths:
             text/plain:
               schema:
                 type: string
-              example: ""
+              example: ''
+  /api/public/v1/verification-job:
+    post:
+      tags:
+        - Verification Job V1
+      summary: Create Verification Job
+      description: >-
+        # Create Verification Job
+
+
+        The create verification job endpoint can match and verify audio or video
+        files against the provided licensing information and any ClicknClear
+        licenses.  
+
+        ClicknClear licenses will be fetched automatically based on the provided
+        email address.
+
+
+        Any songs found in the audio file will be matched against the license
+        files provided.  
+
+        The verification job process is asynchronous, on creation, you will
+        receive the job with an ID, you can use this to poll the Get
+        Verification Job Endpoint or you can set a webhook for the 'Verification
```

Comment:
NIT: is it worth adding preference to these two options e.g. making the webhook option first/adding a "(preferred)" towards the end of the sentence concerning webhooks?

Reply:
Resolved: 5d9413a61706ab45892f02cc55f5cd2966edffe2